### ISSUE TITLE
DiagramsPage: display bug when components in different diagrams have the same ID

### ISSUE THREAD

**Issue by viennem** [ditrit/leto-modelizer#293]
**Describe the bug**

The DiagramsPage has a display bug when components in different diagrams have the same ID.

![image](https://github.com/ditrit/leto-modelizer/assets/20184007/ecfff426-403f-4d8d-b2c4-dcf07f53e158)

There are no errors showing in the console when this bug happens.

**To Reproduce**

1. Start Leto-Modelizer with a tempate library.
   ```bash
   export TEMPLATE_LIBRARY_BASE_URL=https://raw.githubusercontent.com/ditrit/leto-modelizer-templates-library/main
   ```
3. Go to the Home page. 
4. Create an empty project.
5. Create a new diagram with the `AWS S3 Bucket ACL` template.
6. Go to the diagrams page: the diagram is drawn properly.
7. Create a second diagram with the same `AWS S3 Bucket ACL` template.
8. Go to the diagrams page: the second diagram is not drawn properly (see above image). 

**Application**

- leto version `1.2.0`

**Comment by Zorin95670** [comment]
Resolved in #330

**Comment by viennem** [comment]
Reopening because the bug is still happening.
